libs/zd3d12/src/zd3d12.zig

```diff
@@ -51,6 +51,7 @@ pub const GraphicsContext = struct {
     const max_num_buffered_resource_barriers = 16;
 
     device: *d3d12.IDevice9,
+    debug_device: *d3d12.IDebugDevice,
```

Comment:
Let's make it an optional (`?*d3d12.IDebugDevice`) and set it to `null` when debug layer is disabled. In that way users of `zd3d12` module will know if they can use this field.